azure-functions/acbs-function/Dockerfile

```diff
@@ -10,9 +10,7 @@ ENV AzureWebJobsFeatureFlags=EnableWorkerIndexing
 # Directories setup
 WORKDIR /app
 
-COPY libs libs
 COPY azure-functions/acbs-function .
 
-COPY package*.json .
-RUN npm ci
+RUN npm i
```

Comment:
how come we're going from ci -> i? This would worry me a little bit in terms of the package-lock.json not necessarily mapping to what's actually being installed. What people see locally in their package-lock.json compared to what gets built in the dockerfile could be different with this I think

Reply:
ah I see, that sounds like something to be sorted out but not here, approving!

Reply:
There is a ticket `DTFS2-7456` associated with TS upgrade.